### PR TITLE
Dynamic updates devtimes

### PR DESCRIPTION
**Pull request by itzhapp**

### CODE REVIEW

**Comment by itzhapp** [comment]
feat: dynamic update devtimes
mongo db:
+ add collection DevTimes with Username Country Timezone
+ add_dev_time_to_table
+ remove_dev_time_from_table
+ get_all_dev_times

bot commands:
+ add_devtime (staff only)
+ remove_devtime (staff only)

File: src/zorak/cogs/general/general_devtimes.py [hunk 1]
```diff
@@ -30,40 +30,43 @@ async def devtimes(self, ctx):
...
+            embed.add_field(
+                name=f"{dev_time['Country']} ({dev_time['Username']}):",
+                value=f"{tz.strftime('%m/%d/%Y %I:%M %p')}",
+                inline=False,
+            )
         await ctx.respond(embed=embed)
 
+    @commands.slash_command(description="Add entry to devtimes. See pytz.timezone documentation for timezone (Staff Only).")
+    @commands.has_role("Staff")
+    async def add_devtime(self, ctx, username, country, timezone):
```
Comment (Xarlos89) [inline src/zorak/cogs/general/general_devtimes.py:47]: Nice, love that.

File: src/zorak/utilities/core/mongo.py [hunk 2]
```diff
@@ -214,6 +214,43 @@ def initialise_user_table(self):
         self.create_collection("UserPoints", validator_schema=validator)
         logger.info("User table initialised.")
```
Comment (Xarlos89) [inline src/zorak/utilities/core/mongo.py:216]: database, too. very nice

**Review by Xarlos89 — APPROVED** [review]
Looking great
works in test too

**Pull request merged**